test/CMakeLists.txt

```diff
@@ -96,6 +97,7 @@ list( APPEND test_input
 )
 
 list( APPEND test_output
+  testoutput/tropics_20220116.nc4
```

Comment:
@smaticka  is this supposed to be tropics_20220216.nc??? The test is failing because it can't find a file.

Reply:
Yes I think that's it! Thanks @PatNichols